### PR TITLE
DM-42176: Move PeekExposureTask from summit_utils

### PR DESCRIPTION
**Pull request by mfisherlevine**

### CODE REVIEW

**Review by arunkannawadi — APPROVED** [review]

File: tests/test_peekExposureTask.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#
+# LSST Data Management System
+# Copyright 2008-2017 AURA/LSST.
```
Comment (arunkannawadi) [inline tests/test_peekExposureTask.py:3]: This is the old preamble.

File: python/lsst/pipe/tasks/peekExposure.py [hunk 2]
```diff
@@ -0,0 +1,1276 @@
+# This file is part of summit_utils.
```
Comment (arunkannawadi) [inline python/lsst/pipe/tasks/peekExposure.py:1]: Rename to pipe_tasks

File: tests/test_peekExposureTask.py (hunk 1 above)
Comment (mfisherlevine) [reply tests/test_peekExposureTask.py:3]: Huh, it's probably like that in loads of these files then (I just copied and pasted from the quickFrameMeasurement tests as a starting point).

Comment (arunkannawadi) [reply tests/test_peekExposureTask.py:3]: It is outdated there as well.

**Pull request merged**